.github/CODEOWNERS

```diff
@@ -239,6 +239,7 @@ packages/react-components/react-jsx-runtime @microsoft/teams-prg
 packages/react-components/react-toast @microsoft/teams-prg
 packages/react-components/react-search-preview @microsoft/cxe-coastal
 packages/react-components/react-colorpicker-compat @microsoft/cxe-red @sopranopillow
+packages/react-components/react-motion-preview @microsoft/cxe-prg @marcosmoura
```

Comment:
```suggestion
```

This is already declared here https://github.com/microsoft/fluentui/blob/master/.github/CODEOWNERS#L243

Reply:
This was due to a merge conflict. Fixed.